codegenerator/cli/npm/envio/src/PgStorage.res

```diff
@@ -71,10 +71,11 @@ let makeInitializeTransaction = (
 
   let query = ref(
     (
-      reuseExistingPgSchema
+      isEmptyPgSchema && pgSchema === "public"
       // Hosted Service already have a DB with the created public schema
       // It also doesn't allow to simply drop it,
-      // so we reuse an existing schema when it's empty (our case)
+      // so we reuse the existing schema when it's empty
+      // (but only for public, since it's usually always exists)
```

Comment:
I'm not sure when this was introduced but we need to be able to clear past data locally on restart.

Reply:
Please double check this 🙏🏼  and maybe we add a test